lib/mayaUsd/utils/util.cpp

```diff
@@ -1590,7 +1599,23 @@ UsdMayaUtil::setPlugValue(
             }
         }
     }
-    else if (val.IsHolding<VtDoubleArray>()) {
+    else if (val.IsHolding<GfMatrix4d>()) {
```

Comment:
Found by noticing a message from the TF_CODING_ERROR("Unsupported type..." ) from line 1863 in the log of the RenderMan import test. This error message was not sufficient to cause the test to fail though. Should we investigate if that is possible?